packages/search-ui/bin/build-js

```diff
@@ -1,5 +1,8 @@
-# Builds Common JS JavaScript
-babel src --out-dir lib --ignore '**/*.test.js'
+# clear directory
```

Comment:
One of the big changes has been switching out how we build our transpiled code. Before we were using babel to transpile. With this change, we are now using the typescript compiler with configurations for commonjs and esm. I made this choice because it seemed to be the choice for typescript based projects.